dev_tools/conf/pip-list-dev-tools.txt

```diff
@@ -12,8 +12,10 @@ filelock~=3.0.12
 
 # For generating protobufs
 
-# 1.26 is the min version that has python 3.8 wheels
-grpcio-tools~=1.26
+# 1.26.0 is the min version that has python 3.8 wheels
+# however, we can't really go much higher - the protoc version needs to be <= 3.9.2
+# otherwise it will generate pb2 files that are incompatible with tensorflow quantum
+grpcio-tools~=1.26.0
```

Comment:
This seems like a roundabout way of enforcing `protoc <= 3.9.2` - is there any reason we can't just add that as a rule instead? pip-install should ensure compatibility between the selected protoc and grpcio versions.

Reply:
grpcio-tools is an easy way of installing protoc, but it is included into the binary (see this comment for example https://github.com/grpc/grpc/issues/15675#issuecomment-397038811) - so there is no way to enforce protoc with pip.

Reply:
Seems fine to me then.